.vscode/settings.json

```diff
@@ -4,5 +4,11 @@
     "deno.unstable": true,
     "deno.suggest.imports.hosts": {
         "https://deno.land": true
-    }
+    },
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.formatOnSave": true,
+    "workbench.colorTheme": "GitHub Dark",
+    "git.autofetch": true,
+    "gitlens.showWelcomeOnInstall": false,
```

Comment:
```suggestion
```

These three options shouldn't be here, because they override user defaults for everyone.

Reply:
Hey @grebaldi, thanks for the review 🙂 I agree that `workbench.colorTheme` and the `git.autofetch` is definitely really opinionated and we _could_ remove it.

Regarding `gitlens.showWelcomeOnInstall`: I think we should deactivate the welcome page as GitLens gets installed through the extension configuration in the `.gitpod.yml`. So when you throw away a Gitpod workspace and start a new one (e.g. when reviewing a PR), you will see the welcome page again. IMHO this gets annoying pretty quickly 🙂

Reply:
Sure, I don't think anybody would like to keep that setting on `true` anyway, so it's not that intrusive :smile: